ketcher-autotests/tests/utils/canvas/helpers.ts

```diff
@@ -111,10 +111,18 @@ export async function takeTopToolbarScreenshot(page: Page) {
   await expect(editor).toHaveScreenshot();
 }
 
+export async function takePolymerEditorScreenshot(page: Page) {
+  const maxTimeout = 3000;
+  const editor = page.locator('.Ketcher-polymer-editor-root');
+  await waitForRender(page, emptyFunction, maxTimeout);
```

Comment:
JFYI, `waitForRender` just makes no sense as there is no functionality to emit render events in Macromolecules editor.